changelogs/fragments/82-changed_true_action_states_check_mode_yes.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Resource Modules: changed flag is set to true in check_mode for all ACTION_STATES (https://github.com/ansible-collections/ansible.netcommon/pull/82)
```

Oops, colons confuse the yaml parser, need to quote it. Sorry!

```suggestion
  - "Resource Modules: changed flag is set to true in check_mode for all ACTION_STATES (https://github.com/ansible-collections/ansible.netcommon/pull/82)"
```

---

yikes!!! anyway, I have updated this with the suggested change.